bluebrain/repo-bluebrain/packages/coreneuron/package.py

```diff
@@ -104,7 +105,7 @@ class Coreneuron(CMakePackage):
     conflicts('+unified', when='~gpu')
 
     # Older versions do not support GPU execution in shared builds
-    conflicts('+shared', when='@:8.2.0 +gpu')
+    conflicts('+shared', when='@:8.9.9 +gpu')
```

Comment:
This seems fine to me, this support will not be backported to the 8.2 series and probably not to any hypothetical 8.3 series either.